### ISSUE TITLE
GitHub Actions (Workflows) were disabled on Nov 13

### ISSUE THREAD

**Issue by ssbarnea** [ansible/community#505]
It seems that on Nov 13 GitHub Actions were disabled on Ansible organization:

> GitHub Actions workflows can't be executed on this repository.
or on pull requests:

> Workflows can't be executed on this repository. Please check your payment method or billing status.

This directly affected at least two community projects which were using them: [molecule](https://github.com/ansible/molecule/actions) and [ansible-lint](https://github.com/ansible/ansible-lint/actions)

Molecule was not using them for CI but was using them for github automation, including release-note generation.

@gundalow Please re-enabled them ASAP.

**Comment by gundalow** [comment]
Seems to be down for all of github.com/ansible

```
### ERRORED 09:43:31Z

- Workflows can't be executed on this repository. Please check your payment method or billing status.
```

Contacting GitHub support

**Comment by gundalow** [comment]
Looks like it's happening else where:
* https://twitter.com/Ullaakut/status/1100701062095388672
* https://github.community/t5/GitHub-Actions/GitHub-Actions-workflows-can-t-be-executed-on-this-repository/m-p/38153

**Comment by gundalow** [comment]
I've emailed GitHub support

**Comment by mattclay** [comment]
Our account type does not support GitHub Actions now that it has reached GA. We're looking at our options, but I don't have a timeframe for when it may be resolved.

**Comment by dgreene1** [comment]
@lukeautry this happened to us too in tsoa. Do you know what the options are? I believe that Github Actions might require a credit card unless it just locks out when the number of builds are over the free account.

**Comment by webknjaz** [comment]
_/me complains about this on Twitter... https://twitter.com/webKnjaZ/status/1194964915506831362_

**Comment by ssbarnea** [comment]
My guess is twitter (social) is out best friend now, best to reply to their own “marketing” tweets. That is quite a bad move from their side and could negatively impact a huge amount of projects. The only reason why we do not see more of them is because they were smart enough not to be “early adopters”. We experimented, and now we are asked to pay the price🙊

**Comment by friederbluemle** [comment]
If someone else comes across this thread wondering why **GitHub Actions suddenly stopped working** for their open source repos, this might help:

Go to https://github.com/organizations/ORGANIZATION/settings/billing, and

**Switch billing from $0/month to $0/year**

🤦‍♂️

**Comment by gundalow** [comment]
Just to update where we are for Ansible:
1) This only affected github.com/ansible. The other GitHub orgs we have are under the default free open source plan.
2) collection_migration has been migrated to github.com/ansible-community/collection_migration/
3) The only GitHub action used by Molecule was for doing release notes, this has been fixed by using the [Release Drafter](https://github.com/toolmantim/release-drafter/issues/327#issuecomment-554727743) app
4) GitHub *did* email all the GitHub Org admins a few times about this. Though our admins didn't know that we were using GitHub Actions anywhere. Also, there isn't any way to see at an Org level if Actions are used anywhere.
5) github.com/ansible/ansible-lint which also uses Action's hasn't been fixed


So in summary
:heavy_check_mark: collection_migration
:heavy_check_mark: molecule
:negative_squared_cross_mark: : ansible-lint

**Comment by gundalow** [comment]
GitHub actions are now enabled on ansible/ansible thanks to GitHub for sorting this out for is.

**Comment by webknjaz** [comment]
@gundalow is it just ansible/ansible or all public repos under the ansible org?

**Comment by webknjaz** [comment]
FTR I've reenabled it for ansible-lint some time back.

**Comment by gundalow** [comment]
All repos under github.com/ansible
